scripts/init.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 brew install swift-format 
 yarn
+yarn build
```

Comment:
Is this command required here?

Reply:
yes. It generates TS types for library, so tsc will work fine on all projects